data/json/terrain.json

```diff
@@ -5926,6 +6012,224 @@
       ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_glass_railing",
+    "aliases": [ "t_glass_railing_h", "t_glass_railing_v" ],
+    "name": "glass railing",
+    "symbol": "LINE_OXOX",
+    "color": "cyan",
+    "move_cost": 3,
+    "flags": [ "TRANSPARENT", "NOITEM", "THIN_OBSTACLE", "MOUNTABLE", "SHORT", "AUTO_WALL_SYMBOL" ],
+    "connects_to": "RAILING",
+    "bash": {
+      "str_min": 10,
+      "str_max": 40,
+      "sound": "glass braking!",
+      "sound_fail": "ker-rash!.",
+      "ter_set": "t_rock_floor",
+      "items": [
+        { "item": "glass_shard", "prob": 50 },
+        { "item": "pipe", "charges": [ 1, 2 ] },
+        { "item": "scrap", "count": [ 1, 3 ] },
+        { "item": "glass_sheet", "count": [ 1, 2 ] }
```

Comment:
This terrain should probably have a Deconstruct {} that retrieves the glass sheet, and Bash should just create shards.